pkg/templates/charts/toggle/cluster-backup/templates/odap-operator-pre-install-hook.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright Contributors to the Open Cluster Management project
+
+# pre hook used to install the OADP Operator, using the open-cluster-management-backup namespace
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: redhat-oadp-operator-subscription
+  namespace: open-cluster-management-backup
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-1"
+    "helm.sh/resource-policy": delete
+spec:
+  channel: stable-1.1
+  config:
+    resources: {}
+{{- with .Values.hubconfig.nodeSelector }}
+    nodeSelector:
+{{ toYaml . | indent 6 }}
+{{- end }}
+{{- with .Values.hubconfig.tolerations }}
+    tolerations:
+    {{- range . }}
+    - {{ if .Key }} key: {{ .Key }} {{- end }}
+      {{ if .Operator }} operator: {{ .Operator }} {{- end }}
+      {{ if .Value }} value: {{ .Value }} {{- end }}
+      {{ if .Effect }} effect: {{ .Effect }} {{- end }}
+      {{ if .TolerationSeconds }} tolerationSeconds: {{ .TolerationSeconds }} {{- end }}
+      {{- end }}
+{{- end }}
+    {{- if .Values.hubconfig.proxyConfigs }}
+    env:
+    - name: HTTP_PROXY
+      value: {{ .Values.hubconfig.proxyConfigs.HTTP_PROXY }}
+    - name: HTTPS_PROXY
+      value: {{ .Values.hubconfig.proxyConfigs.HTTPS_PROXY }}
+    - name: NO_PROXY
+      value: {{ .Values.hubconfig.proxyConfigs.NO_PROXY }}
+    {{- end }}
+  installPlanApproval: Automatic
+  name: redhat-oadp-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
These have been hardcoded where [before](https://github.com/stolostron/multiclusterhub-repo/blob/main/multiclusterhub/charts/cluster-backup-chart/templates/odap-operator-pre-install-hook.yaml#L35-L38) they were configurable

Reply:
@JakobGray I hardcoded them because I was under the impression that they were just being drawn from the values.yaml. Do we want to add them to the things that are applied by the render? I'm not sure where in our code they would be configured